content/blog/2020-05-dplyr-1-0-0-last-minute-additions/index.Rmarkdown

```diff
@@ -0,0 +1,147 @@
+---
+title: 'dplyr 1.0.0: last minute additions'
+author: Hadley Wickham
+date: '2020-05-06'
+slug: dplyr-1-0-0-last-minute-additions
+categories:
+  - package
+tags:
+  - dplyr
+description: >
+    Learn about two last minute additions to dplyr 1bl.0.0: a chattier 
```

Comment:
```suggestion
    Learn about two last-minute additions to dplyr 1.0.0: a chattier 
```

Reply:
Kirill's catch, plus hyphenated "last minute," since it's modifying "additions."